custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -236,7 +236,8 @@ def is_text(x):
         """
         Decide if a pandas series is text, using a very simple heuristic:
         1. Count the number of elements in the series that contain 1 or more whitespace character
-        2. If >75% of the elements have whitespace, the Series is text
+        2. If >75% of the elements have whitespace, the Series is text and there are more than 60 unique values or
+            more than 5% of values are unique
```

Comment:
@scottp-ml I'm not sure I understand this comment. I believe it's saying:

"The series is determined to be text IF >75% of the elements have whitespace and (there are more than 60 unique values or more than 5% of values are unique)"

Is that correct?

Reply:
yes please rewrite for clarity